### ISSUE TITLE
Unable to compile for GIMP on mac

### ISSUE THREAD

**Issue by FrostBird347** [pfusik/qoi-ci#7]
Trying to run `make install-gimp` on mac quickly fails with this output:

```
cc -O2 -Wall -I transpiled `gimptool-2.0 --cflags` -o file-qoi file-qoi.c QOI-stdio.c transpiled/QOI.c -rpath @executable_path/../../../ -L /Applications/GIMP-2.10.app/Contents/Resources/lib -lgimpui-2.0.0 -lgimp-2.0.0 -lgegl-0.4.0 -lbabl-0.1.0 -lgobject-2.0.0
Package gimpui-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gimpui-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gimpui-2.0' found
No output from 'pkg-config --cflags gimpui-2.0'
file-qoi.c:27:10: fatal error: 'libgimp/gimp.h' file not found
#include <libgimp/gimp.h>
         ^~~~~~~~~~~~~~~~
1 error generated.
make: *** [file-qoi] Error 1
```

Downloading gimp's source and copying the libgimp folder over produces this output:

```
cc -O2 -Wall -I transpiled `gimptool-2.0 --cflags` -o file-qoi file-qoi.c QOI-stdio.c transpiled/QOI.c -rpath @executable_path/../../../ -L /Applications/GIMP-2.10.app/Contents/Resources/lib -lgimpui-2.0.0 -lgimp-2.0.0 -lgegl-0.4.0 -lbabl-0.1.0 -lgobject-2.0.0
Package gimpui-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gimpui-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gimpui-2.0' found
No output from 'pkg-config --cflags gimpui-2.0'
file-qoi.c:27:10: error: 'libgimp/gimp.h' file not found with <angled> include; use "quotes" instead
#include <libgimp/gimp.h>
         ^~~~~~~~~~~~~~~~
         "libgimp/gimp.h"
In file included from file-qoi.c:27:
./libgimp/gimp.h:24:10: fatal error: 'cairo.h' file not found
#include <cairo.h>
         ^~~~~~~~~
2 errors generated.
make: *** [file-qoi] Error 1
```

I can place quotes around `libgimp/gimp.h` and `libgimp/gimpui.h`, however that still does not fix `'cairo.h' file not found` 
I have also verified that cairo has already been installed via both homebrew and macports

**Comment by FrostBird347** [comment]
Unfortunately, it seems that I have `cairo` installed via both homebrew and macports, yet neither are being found.

As for `gimpui-2.0`, I have been unable to find any homebrew or macports packages that start with `gimpui`

**Comment by FrostBird347** [comment]
It turns out that I had symlinked gimptool-2.0 from the application (`/Applications/GIMP-2.10.app/Contents/MacOS/gimptool-2.0`) to a bin folder,
Removing that symlink and reinstalling gimp via macports again fixed all these errors and allowed me to compile it.
(However, the macports version of gimp would stop responding before the UI appeared)

While it is able to open qoi images, attempting to save a qoi image fails with this error: 
![export](https://user-images.githubusercontent.com/39435218/171409649-cf238eb0-045c-490d-aebc-c2ad973c8a9c.png)

Opening gimp through terminal and looking at the output shows these logs on each qoi save attempt:
```
objc[70939]: Class GNotificationCenterDelegate is implemented in both /opt/local/lib/libgio-2.0.0.dylib (0x10c7fc8f8) and /usr/local/lib/libgio-2.0.0.dylib (0x10d5cbc58). One of the two will be used. Which one is undefined.

(process:70939): GLib-GObject-WARNING **: **:00:14.446: invalid cast from '(null)' to 'GTypeModule'

(process:70939): GLib-GObject-WARNING **: **:00:14.447: plugin pointer (0x7fc26d004d20) for type 'GTlsBackendGnutls' is invalid

(process:70939): GLib-GObject-CRITICAL **: **:00:14.447: g_type_add_interface_dynamic: assertion 'G_TYPE_IS_INSTANTIATABLE (instance_type)' failed

(process:70939): GLib-GIO-WARNING **: **:00:14.447: Tried to implement non-registered extension point gio-tls-backend

(file-qoi:70939): Gtk-WARNING **: **:00:14.450: cannot open display: frostbird347.******.******
/Applications/GIMP-2.10.app/Contents/MacOS/gimp: LibGimpBase-WARNING: gimp: gimp_wire_read(): error
```


<details>
  <summary>In addition to that, loading a qoi file produces these errors: (<i>click to expand</i>)</summary>
  

```
objc[74725]: Class GNotificationCenterDelegate is implemented in both /opt/local/lib/libgio-2.0.0.dylib (0x1014818f8) and /usr/local/lib/libgio-2.0.0.dylib (0x102261c58). One of the two will be used. Which one is undefined.

(process:74725): GLib-GObject-WARNING **: **:05:49.073: invalid cast from '(null)' to 'GTypeModule'

(process:74725): GLib-GObject-WARNING **: **:05:49.074: plugin pointer (0x7fbf2b004120) for type 'GTlsBackendGnutls' is invalid

(process:74725): GLib-GObject-CRITICAL **: **:05:49.076: g_type_add_interface_dynamic: assertion 'G_TYPE_IS_INSTANTIATABLE (instance_type)' failed

(process:74725): GLib-GIO-WARNING **: **:05:49.077: Tried to implement non-registered extension point gio-tls-backend
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/ycbcr.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/ycbcr.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/fast-float.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/fast-float.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/HCY.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/HCY.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse-half.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse-half.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/grey.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/grey.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse2-int8.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse2-int8.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/avx2-int8.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/avx2-int8.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gegl-fixups.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gegl-fixups.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/simple.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/simple.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl-table-lies.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl-table-lies.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/CIE.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/CIE.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl-lies.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl-lies.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/double.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/double.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse2-int16.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse2-int16.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/float.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/float.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/cairo.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/cairo.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/HSL.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/HSL.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/u16.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/u16.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/u32.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/u32.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/half.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/half.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gimp-8bit.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gimp-8bit.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/HSV.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/HSV.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl-table.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/gggl-table.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse4-int8.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse4-int8.dylib
  Reason: image not found
../babl-0.1.92/babl/babl-internal.h:214 void babl_log(const char *, ...)()
	dlopen() failed:
	dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse2-float.dylib, 2): Library not loaded: @rpath/libbabl-0.1.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/babl-0.1/sse2-float.dylib
  Reason: image not found
GEGL-Message: **:05:49.126: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/seamless-clone.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/seamless-clone.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/seamless-clone.dylib
  Reason: image not found
GEGL-Message: **:05:49.126: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pixbuf-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pixbuf-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pixbuf-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.126: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common-gpl3.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common-gpl3.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common-gpl3.dylib
  Reason: image not found
GEGL-Message: **:05:49.126: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-core.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-core.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-core.dylib
  Reason: image not found
GEGL-Message: **:05:49.127: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/png-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/png-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/png-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.127: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/jpg-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/jpg-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/jpg-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.128: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/tiff-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/tiff-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/tiff-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.128: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/rgbe-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/rgbe-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/rgbe-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.128: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/svg-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/svg-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/svg-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.129: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/vector-stroke.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/vector-stroke.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/vector-stroke.dylib
  Reason: image not found
GEGL-Message: **:05:49.129: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pixbuf-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pixbuf-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pixbuf-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.130: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/png-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/png-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/png-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.130: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/jpg-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/jpg-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/jpg-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.131: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/tiff-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/tiff-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/tiff-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.131: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/rgbe-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/rgbe-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/rgbe-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.132: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/exr-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/exr-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/exr-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.132: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/seamless-clone-compose.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/seamless-clone-compose.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/seamless-clone-compose.dylib
  Reason: image not found
GEGL-Message: **:05:49.133: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common-cxx.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common-cxx.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common-cxx.dylib
  Reason: image not found
GEGL-Message: **:05:49.133: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/lcms-from-profile.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/lcms-from-profile.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/lcms-from-profile.dylib
  Reason: image not found
GEGL-Message: **:05:49.134: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-generated.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-generated.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-generated.dylib
  Reason: image not found
GEGL-Message: **:05:49.134: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/text.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/text.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/text.dylib
  Reason: image not found
GEGL-Message: **:05:49.135: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gif-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gif-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gif-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.135: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/npd.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/npd.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/npd.dylib
  Reason: image not found
GEGL-Message: **:05:49.136: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/ppm-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/ppm-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/ppm-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.136: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/npy-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/npy-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/npy-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.137: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/webp-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/webp-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/webp-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.137: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pdf-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pdf-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/pdf-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.138: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/exr-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/exr-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/exr-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.138: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/transformops.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/transformops.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/transformops.dylib
  Reason: image not found
GEGL-Message: **:05:49.139: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/gegl-common.dylib
  Reason: image not found
GEGL-Message: **:05:49.139: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/ppm-save.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/ppm-save.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/ppm-save.dylib
  Reason: image not found
GEGL-Message: **:05:49.140: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/path.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/path.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/path.dylib
  Reason: image not found
GEGL-Message: **:05:49.140: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/webp-load.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/webp-load.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/webp-load.dylib
  Reason: image not found
GEGL-Message: **:05:49.140: Module '/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/matting-levin.dylib' load error: dlopen(/Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/matting-levin.dylib, 10): Library not loaded: @rpath/libgegl-0.4.0.dylib
  Referenced from: /Applications/GIMP-2.10.app/Contents/Resources/lib/gegl-0.4/matting-levin.dylib
  Reason: image not found
```

  </details>


I tried uninstalling the macports version of gimp, verified that it couldn't compile and launched the standalone app with the plugin installed. It was unable to open or save any qoi images due to `/opt/local/lib/libgimpui-2.0.0.dylib` being deleted. 
Finally, I installed the homebrew version of gimp to find no `gimptool-2.0`.

**Comment by pfusik** [comment]
This looks to me like you are affected by conflicting installations of GIMP.

I used homebrew exclusively for the build. I think `gimptool-2.0` should be installed as part of `libgimp2.0`. Run `otool -L file-qoi` to see the dynamic link dependencies.

`make install-gimp` installs the plugin for the [native build](https://www.gimp.org/downloads/) of GIMP (i.e., not homebrew's or macports). You might run into an error because of the security mechanisms I mentioned - run GIMP from the terminal to see the messages.

Also, what's your OS version and architecture (M1/Intel) ?

**Comment by pfusik** [comment]
I've released [1.1.2](https://github.com/pfusik/qoi-ci/releases/tag/qoi-ci-1.1.2) with the macOS GIMP plugin.

**Comment by FrostBird347** [comment]
Thanks.
It took far long for me to check, however I can confirm that the released plugin works correctly.
